jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -969,6 +985,26 @@ public void testGetJavaContainerBuilderWithBaseImage_incompatibleJava8BaseImage(
                     rawConfiguration, projectProperties, inferredAuthProvider));
     assertThat(exception2.getBaseImageMajorJavaVersion()).isEqualTo(8);
     assertThat(exception2.getProjectMajorJavaVersion()).isEqualTo(11);
+
+    when(rawConfiguration.getFromImage()).thenReturn(Optional.of("eclipse-temurin:8"));
+    IncompatibleBaseImageJavaVersionException exception3 =
+        assertThrows(
+            IncompatibleBaseImageJavaVersionException.class,
+            () ->
+                PluginConfigurationProcessor.getJavaContainerBuilderWithBaseImage(
+                    rawConfiguration, projectProperties, inferredAuthProvider));
+    assertThat(exception3.getBaseImageMajorJavaVersion()).isEqualTo(8);
+    assertThat(exception3.getProjectMajorJavaVersion()).isEqualTo(11);
+
+    when(rawConfiguration.getFromImage()).thenReturn(Optional.of("eclipse-temurin:8-jre"));
+    IncompatibleBaseImageJavaVersionException exception4 =
+        assertThrows(
+            IncompatibleBaseImageJavaVersionException.class,
+            () ->
+                PluginConfigurationProcessor.getJavaContainerBuilderWithBaseImage(
+                    rawConfiguration, projectProperties, inferredAuthProvider));
+    assertThat(exception4.getBaseImageMajorJavaVersion()).isEqualTo(8);
+    assertThat(exception4.getProjectMajorJavaVersion()).isEqualTo(11);
```

Comment:
This test seems to be getting a little long. Can we have this in a separate test or make it parameterized? Same below.